lib/constants/payment-methods.js

```diff
@@ -30,3 +30,9 @@ const matchPm = (pm, service, type) => {
 export const isPrepaid = pm => {
   return matchPm(pm, PaymentMethodService.OPEN_COLLECTIVE, PaymentMethodType.PREPAID);
 };
+
+/** Returns payment methods with recurring payment support **/
+export const ProvidersWithRecurringPaymentSupport = [
+  GQLV2_PAYMENT_METHOD_TYPES.CREDIT_CARD,
+  GQLV2_PAYMENT_METHOD_TYPES.PAYPAL,
```

Comment:
Technically, `ACCOUNT_BALANCE` is part of this list since it supports recurring too. But I understand this can require more changes to your code above, so we can keep it like that for now. A comment could eventually be helpful to make sure someone won't just add the value to the array without being aware of the side-effect

Reply:
@Betree : Hmm.... Sorry, I don't quite understand here why adding account balance involve side effects? 🤔 🤔

Reply:
I meant that it probably impacts the places where you're using `ProvidersWithRecurringPaymentSupport`. But if it's not too difficult to change, I think it makes sense to have the account balance in this array.

Reply:
I don't think this is a problem at all, so I've went ahead and added the type. 👍🏽